### PR TITLE
[BUGFIX] tab switch bug

### PR DESCRIPTION
**Pull request by xandrej**
__Tasks:__
- [x] done
- [ ] todo
- [ ] got feedback

__Informations_: Bug fix - when you switch tabs saved data is not shown 

| Q            | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Tests pass?  | yes
| Fixed tickets | https://github.com/massiveart/POOL-ALPIN/issues/1218, https://github.com/sulu-io/SuluProductBundle/issues/157
| Doc          | none

### CODE REVIEW

File: Resources/public/js/components/bulk-price/main.js [hunk 1]
```diff
@@ -18,14 +18,7 @@ define(['text!suluproduct/components/bulk-price/bulk-price.html'], function(Bulk
...
 
-    var defaults = {
-            instanceName: null,
-            data: [],
-            translations: {},
-            currency: null
-        },
-
-        constants = {
+        var constants = {
```
Comment (michaelzangerle) [inline Resources/public/js/components/bulk-price/main.js]: check formatting

File: Resources/public/js/components/bulk-price/main.js [hunk 2]
```diff
@@ -18,14 +18,7 @@ define(['text!suluproduct/components/bulk-price/bulk-price.html'], function(Bulk
 
     'use strict';
 
-    var defaults = {
```
Comment (michaelzangerle) [inline Resources/public/js/components/bulk-price/main.js]: why did you remove the defaults? do you need none of them?

Comment (xandrej) [inline Resources/public/js/components/bulk-price/main.js]: because the variable is not used

File: Resources/public/js/components/bulk-price/main.js [hunk 3]
```diff
@@ -180,23 +197,26 @@ define(['text!suluproduct/components/bulk-price/bulk-price.html'], function(Bulk
...
                 salesPrice = getSalesPriceAndRemoveFromPrices.call(this, prices);
             }
-
-            if (this.options.data.specialPrice) {
-                specialPrice = this.options.data.specialPrice;
-                specialPrice.price = this.sandbox.numberFormat(specialPrice.price, 'n');
+            
+            if (this.options.data.attributes.specialPrices) {
+                specialPrice = getSpecialPriceForCurrency.call(this, this.options.data.attributes.specialPrices, currencyCode);
+                if (specialPrice) {
```
Comment (michaelzangerle) [inline Resources/public/js/components/bulk-price/main.js]: is this check needed?

**Comment by michaelzangerle** [comment]
else ACK
